### PR TITLE
alphalens from quantopian

### PR DESCRIPTION
**Pull request by alen12345**
I have already opened a PR in the upstream to include the LICENSE :)

### CODE REVIEW

File: recipes/alphalens/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+about:
+  home: http://quantopian.github.io/alphalens/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'Performance analysis of predictive (alpha) stock factors'
+
+  description: |
+    Alphalens is a Python Library for performance analysis of predictive (alpha) stock factors. Alphalens works great with the Zipline open source backtesting library, and Pyfolio which provides performance and risk analysis of financial portfolios.
```
Comment (CJ-Wright) [inline recipes/alphalens/meta.yaml]: Could you please break this up over multiple lines?

Comment (alen12345) [reply recipes/alphalens/meta.yaml]: Yes sure!

**Comment by alen12345** [comment]
@CJ-Wright if you want any other formatting to be edited, please let me know :)

**Comment by CJ-Wright** [comment]
LGTM

**Pull request merged**